arches/app/src/components/ControlledListManager/ImageEditor.vue

```diff
@@ -0,0 +1,114 @@
+<script setup lang="ts">
+import arches from "arches";
+import { computed, inject } from "vue";
+import { useGettext } from "vue3-gettext";
+
+import Button from "primevue/button";
+import { useToast } from "primevue/usetoast";
+
+import AddMetadata from "@/components/ControlledListManager/AddMetadata.vue";
+import ImageMetadata from "@/components/ControlledListManager/ImageMetadata.vue";
+
+import { DANGER, METADATA_CHOICES, itemKey } from "@/components/ControlledListManager/constants.ts";
+import { deleteImage } from "@/components/ControlledListManager/api.ts";
+import { bestLabel } from "@/components/ControlledListManager/utils.ts";
+
+import type { Ref } from "vue";
+import type {
+    ControlledListItem,
+    ControlledListItemImage,
+} from "@/types/ControlledListManager";
+
+const item = inject(itemKey) as Ref<ControlledListItem>;
+const { image } = defineProps<{ image: ControlledListItemImage }>();
+
+const toast = useToast();
+const { $gettext } = useGettext();
+
+const labeledChoices = [
+    {
+        type: METADATA_CHOICES.title,
+        label: $gettext('Title'),
+    },
+    {
+        type: METADATA_CHOICES.alternativeText,
+        label: $gettext('Alternative text'),
+    },
+    {
+        type: METADATA_CHOICES.description,
+        label: $gettext('Description'),
+    },
+    {
+        type: METADATA_CHOICES.attribution,
+        label: $gettext('Attribution'),
+    },
+];
+
+const bestTitle = computed(() => {
```

Comment:
This function checks the titles and prefers the one in the active language, but now that I'm looking at it, it should fallback to something else if it can.